<launch>

  <node pkg="trajectory_generator" type="trajectory_generator_node" name="trajectory_generator_node" output="screen" required = "true">

      <remap from="~waypoints" to="/waypoint_generator/waypoints"/>
      <remap from="~odom"      to="/msckf_vio/odometry"/>
      <remap from="~joy"      to="/djiros/rc"/>   
      <param name="planning/vel"       value="2.0"  />
      <param name="planning/acc"       value="2.0"  />
      <param name="planning/der_order" value="4"    />
      <param name="planning/min_order" value="3"    />
      <param name="vis/vis_traj_width" value="0.05" />
  </node>
  <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log" args="-d $(find trajectory_generator)/launch/rviz_config/test_traj.rviz" />

</launch>
